openstack/castellum/templates/_utils.tpl

```diff
@@ -11,8 +11,13 @@
   value: "false"
 - name: CASTELLUM_ASSET_MANAGERS
   value: "nfs-shares,project-quota"
-- name: CASTELLUM_DB_URI
-  value: "postgres://postgres:{{ .Values.postgresql.postgresPassword }}@castellum-postgresql.{{ .Release.Namespace }}.svc/castellum?sslmode=disable"
+- name: CASTELLUM_DB_PASSWORD
+  valueFrom:
+    secretKeyRef:
+      name: castellum-secret
+      key: postgres_password
+- name: CASTELLUM_DB_HOSTNAME
+  value: "castellum-postgresql.{{ .Release.Namespace }}.svc"
```

Comment:
Relating to my review comment on the code PR, this section needs an explicit mention of `sslmode=disable` since this value should not be the default for connection options.